app/static/app/js/components/EditTaskForm.jsx

```diff
@@ -85,19 +86,16 @@ class EditTaskForm extends React.Component {
 
   loadProcessingNodes(){
     const failed = () => {
-      this.setState({error: "Could not load list of processing nodes. Are you connected to the internet?"});
+      this.setState({error: _("Could not load list of processing nodes. Are you connected to the internet?")});
     }
 
     this.nodesRequest = 
       $.getJSON("/api/processingnodes/?has_available_options=True", json => {
         if (Array.isArray(json)){
           // No nodes with options?
           const noProcessingNodesError = (nodes) => {
-            var extra = nodes ? "We tried to reach:<ul>" + nodes.map(n => Utils.html`<li><a href="${n.url}">${n.label}</a></li>`).join("") + "</ul>" : "";
-            this.setState({error: `There are no usable processing nodes. ${extra}Make sure that at least one processing node is reachable and
-             that you have granted the current user sufficient permissions to view 
-             the processing node (by going to Administration -- Processing Nodes -- Select Node -- Object Permissions -- Add User/Group and check CAN VIEW PROCESSING NODE).
-             If you are bringing a node back online, it will take about 30 seconds for WebODM to recognize it.`});
+            var extra = nodes ? _("We tried to reach:") + "<ul>" + nodes.map(n => Utils.html`<li><a href="${n.url}">${n.label}</a></li>`).join("") + "</ul>" : "";
+            this.setState({error: _("There are no usable processing nodes.") + extra + _("Make sure that at least one processing node is reachable and that you have granted the current user sufficient permissions to view the processing node (by going to Administration -- Processing Nodes -- Select Node -- Object Permissions -- Add User/Group and check CAN VIEW PROCESSING NODE). If you are bringing a node back online, it will take about 30 seconds for WebODM to recognize it.")});
```

Comment:
```suggestion
            this.setState({error: _("There are no usable processing nodes.") + extra + _("Make sure that at least one processing node is reachable and that you have granted the current user sufficient permissions to view the processing node (by going to Administration -- Processing Nodes -- Select Node -- Object Permissions -- Add User/Group and check CAN VIEW PROCESSING NODE). If you are bringing a node back online, it will take about 30 seconds for WebODM to recognise it.")});
```